api-samples/tabs/inspector/tabs_api.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tabs Inspector</title>
+    <link rel="stylesheet" href="./tabs_api.css" />
+  </head>
+
+  <body>
+    <div id="windowList">
+      <!-- WindowItems Here -->
+    </div>
+    <template id="windowItem">
+      <div class="window-item">
+        <div class="window-id-wrapper">
+          Window: <span class="window_id"></span>
+        </div>
+        <div class="window-status-wrapper">
+          left:
+          <input class="window_left" type="text" /> top:
+          <input class="window_top" type="text" /> width:
+          <input class="window_width" type="text" /> height:
+          <input class="window_height" type="text" />
+          <input class="window_focused" type="checkbox" disabled /> Focused
+          <input class="window_current" type="checkbox" disabled /> Current
+          <button class="window_refresh">Refresh</button>
+        </div>
+        <div id="tabList">
+          <!-- TabItems Here -->
+        </div>
+        <button class="update_window_button">Update Window</button>
+        <button class="remove_window_button">Close Window</button>
+        <button class="refresh_active_tab_button">Refresh Active Tab</button>
+      </div>
+    </template>
+    <template id="tabItem">
+      <div class="tab-item">
+        <div class="wrapper">
+          <div class="tab_id"></div>
+          <div class="tab-actions-wrapper">
+            index:
+            <input type="text" class="tab_index" />
+            windowId:
+            <input type="text" class="tab_window_id" />
+            <button class="move_tab_button">Move</button>
+            <button class="refresh_tab_button">Refresh</button>
+          </div>
+        </div>
+        <div class="wrapper">
+          <div class="input-group">
+            <div class="label">title:</div>
+            <input type="text" class="tab_title" />
+          </div>
+          <div class="input-group">
+            <div class="label">url:</div>
+            <input type="text" class="tab_url" />
+          </div>
+          <div>
+            <input type="checkbox" class="tab_active" disabled /> Active
```

Comment:
So I realise I suggested making this disabled but I'm actually not sure. I think we should either:

- Remove `disabled`, but have it where when you check one it unchecks all of the others for that window
- Change `updateTabData` to no longer set `active`, since it isn't ever going to change

That way both bits of code make sense :)